src/web/components/StickyNav/LazyExpandedMenu.tsx

```diff
@@ -0,0 +1,146 @@
+import React, { useEffect, useState } from 'react';
+import { css } from 'emotion';
+
+import { brandBackground } from '@guardian/src-foundations/palette';
+import { textSans } from '@guardian/src-foundations/typography';
+import { from, until } from '@guardian/src-foundations/mq';
+
+import { Display } from '@guardian/types';
+import {
+	buildID,
+	navInputCheckboxId,
+} from '@root/src/web/components/Nav/config';
+import { extractNAV } from '@root/src/model/extract-nav';
+
+import { Columns } from './ExpandedMenu/Columns';
+import { ShowMoreMenu } from './ExpandedMenu/ShowMoreMenu';
+import { VeggieBurgerMenu } from './ExpandedMenu/VeggieBurgerMenu';
+
+const mainMenuStyles = (ID: string) => css`
+	background-color: ${brandBackground.primary};
+	box-sizing: border-box;
+	${textSans.large()};
+	left: 0;
+	margin-right: 29px;
+	padding-bottom: 24px;
+	top: 0;
+	z-index: 1070;
```

Comment:
I remember having issues with `z-index` on the Nav bar, but have you tried using our zIndex function? (I think its unlikely to work but worth a shot)

Reply:
Oh @liywjl that is something I missed. I am using the helper function elsewhere and should have updated this too. Thanks.